src/rct/cert_commands.py

```diff
@@ -86,19 +86,19 @@ def __init__(self):
     def _do_command(self):
         cert = self._create_cert()
         pem = self._get_pem(self._get_file_from_args())
-        print(_("Type: %s") % type_to_string(cert))
-        print(_("Version: %s") % cert.version)
+        print(_("Type: {certificate_type}").format(certificate_type=type_to_string(cert)))
+        print(_("Version: {certificate_version}").format(certificate_version=cert.version))
         print(_("DER size: %db") % self._get_der_size(pem))
```

Comment:
this was fine to change btw, it needs a `{size}` or similar placeholder

Reply:
I changed it back to use format with a `{size}` placeholder